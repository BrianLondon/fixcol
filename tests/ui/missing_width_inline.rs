use fixed_derive::{ReadFixed};

#[derive(ReadFixed)]
struct Foo(u64);

pub fn main() {}
