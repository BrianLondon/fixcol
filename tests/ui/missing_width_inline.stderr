error: fixed-derive error: Width must be specified for all fields.
 --> tests/ui/missing_width_inline.rs:4:12
  |
4 | struct Foo(u64);
  |            ^^^
